model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/resource_utilization_data.py

```diff
@@ -132,6 +133,54 @@ def compute_nodes_weights_params(graph: Graph, fw_info: FrameworkInfo) -> Tuple[
 
     return np.array(weights_memory_bytes), np.array(weights_params)
 
+
+def compute_activation_output_maxcut_sizes(graph: Graph) -> Tuple[np.ndarray, np.ndarray]:
+    """
+    Computes an array of the respective output tensor maxcut size and an array of the output tensor
+    cut size in bytes for each cut.
+
+    Args:
+        graph: A finalized Graph object, representing the model structure.
+
+    Returns:
+    A tuple containing two arrays:
+        - The first is an array of the size of each activation max-cut size in bytes, calculated
+          using the maximal bit-width for quantization.
+        - The second array an array of the size of each activation max-cut activation size in number of parameters.
+
+    """
+    cuts = calc_graph_cuts(graph)
+
+    # map nodes to cuts.
+    node_to_cat_mapping = {}
+    for i, cut in enumerate(cuts):
+        mem_element_names = [m.node_name for m in cut.mem_elements.elements]
+        for m_name in mem_element_names:
+            if len(graph.find_node_by_name(m_name)) > 0:
+                if m_name in node_to_cat_mapping:
+                    node_to_cat_mapping[m_name].append(i)
+                else:
+                    node_to_cat_mapping[m_name] = [i]
+            else:
+                raise Exception("Missing node")
+
+    activation_outputs = np.zeros(len(cuts))
+    activation_outputs_bytes = np.zeros(len(cuts))
+    for n in graph.nodes:
+        # Go over all nodes that have configurable activation.
```

Comment:
The comment says configurable (i.e. >1 q candidate), the code says any quantizable